Nighttime-Operations/Weather-Constraints/index.rst

```diff
@@ -0,0 +1,179 @@
+        Before deciding to open again, ensure that sustained winds have dropped below the closure limits for more than 15 minutes. 
+
+.. _weather-constraints-humidity-and-dew-point:
+
+Humidity and dew point
+^^^^^^^^^^^^^^^^^^^^^^
+
+Higher relative humidity is not an absolute determination of closure, but observers must pay close attention when humidity is over 70%, or if it begins to suddenly trend upward.
+
+Humidity above 65%:
+    - Turn off the extraction fan and close vent gates.
+
+Humidity above 70%:
+    - Observers must treat this range with extra caution. 
+        Take time to go outside and inspect vehicles, buildings, and look for falling drops from the roof. 
+        Be mindful of local humidity changes between AuxTel and the main telescope building.
+        Because AuxTel is at a slightly higher elevation, there is a small buffer against condensation.
+        If the ambient temperature is cooler, especially during winter time, the condensation risk on metal surfaces is higher. 
+
+The quantity that is more important than relative humidity is the **dew point temperature** and the **dew point difference**. 
+
+The dew point temperature is the ambient air temperature at which relative humidity will reach 100% - the air is completely saturated with water vapor.
+
+The dew point difference is the difference between the dew point and the coldest structure in the telescope dome. 
+
+Example:
+    Dew point temperature = 8 degrees C.
+
+    AuxTel M1 temperature = 6 degrees C.
+    
+    Glycol lines = 4 degrees C.
+    
+    Dew point difference = 4 degrees C.
+
+For any object or strucutre that is above the dew point temperature, there is less condensation risk.
+Water condenses on the coolest surface. 
+
+Dew point difference is > 2.5 degrees C:
+    - Normal operations may continue.
+
+Dew point difference is between 2 and 2.5 degrees C:
+    - Observers must consider closing. 
+        Go outside and check condensation on buildings and vehicles. 
+        If wet, advise to close.
+
+Dew point difference is 2 degrees C or less:
+    - Close immediately. 
+
+Observers must wait for the dew point temperature or humidity to recover from the closure limits for at least 15 minutes before opening again. 
+
+.. note::
+    If humidity has dropped during the night, before opening the dome slit, consider performing a quick dome drain procedure. 
+    Rotate the closed dome several times in both directions, exercisicing large slews. 
+    This can remove some of the water droplets that might have accumulated on the metal surface. 
+
+.. _weather-constraints-cloud-cover:
+
+Cloud cover
+^^^^^^^^^^^
+
+Until DIMM flux measurements are imported into Rubin TV or in the EFD, or the wavefront sensors of LSSTCam are available, determining safe or unsafe cloud coverage is a bit of an un-tested process.
+
+AuxTel has continued observing spectrae happily in 2.5 magnitudes of extinction, given that most targets are typically <= 8 magnitudes.
```

Comment:
is spectrae a word? If so then it's awesome.

I think you can just drop the word as we do both imaging and spectra. LSST will also observe in junky conditions... it's not like we can switch instruments!

Reply:
👍 It is not a word. lol